test/test_utils.go

```diff
@@ -218,3 +223,54 @@ func printDebugging(t *testing.T, ns, taskRunName string) {
 	output, _ = exec.Command("kubectl", "logs", "deploy/tekton-chains-controller", "-n", "tekton-chains").CombinedOutput()
 	t.Log(string(output))
 }
+
+func verifySignature(ctx context.Context, t *testing.T, c *clients, tr *v1beta1.TaskRun) {
+	// Retrieve the configuration.
+	chainsConfig, err := c.KubeClient.CoreV1().ConfigMaps("tekton-chains").Get(ctx, "chains-config", metav1.GetOptions{})
+	if err != nil {
+		t.Errorf("error retrieving tekton chains configmap: %s", err)
+	}
+	cfg, err := config.NewConfigFromConfigMap(chainsConfig)
+	if err != nil {
+		t.Errorf("error creating tekton chains configuration: %s", err)
+	}
+
+	// Prepare the logger.
+	logger, _ := zap.NewProduction()
```

Comment:
there's a testing logger we can use [here](https://github.com/tektoncd/chains/blob/24c560f488ddc2b60bb6a4d2d6059ceea2db90db/pkg/chains/formats/intotoite6/intotoite6_test.go#L392)